modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/AutoDownloadSettingsPodcastsPage.kt

```diff
@@ -88,9 +89,10 @@ internal fun AutoDownloadSettingsPodcastsPage(
         }
 
         Spacer(
-            modifier = Modifier.height(16.dp),
+            modifier = Modifier.height(8.dp),
         )
         FadedLazyColumn(
+            fadeConfig = FadeConfig.Default.copy(showEndFade = false),
```

Comment:
While looking at the auto download choose page, I noticed it has faded at the bottom of the list. It doesn't look quite right as we don't do this elsewhere, so I have removed it.